### PR TITLE
Several fixes to make the tests pass

### PR DESCRIPTION
**Pull request by tcbyrd**
This PR contains several small fixes to make the tests pass. A couple of changes were needed:

- Now that we have an option to disable `/probot/stats`, it's no longer a reliable way to check the status of an app. Instead this just hits `/ping`.
- ☝️ Uncovered that `untrivializer` is not being maintained, so I removed it. The glitch app is throwing a 503. Remixing it showed that it hadn't been updated since Probot 3.0.
- Finally, the html-proofer script was failing on `https://localhost:3000` from the docs page, so this updates the ignored urls to address that

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -21,9 +21,13 @@ <h1 class="alt-h2 text-center">Join the Community</h1>
       Contribute on GitHub
     </a>
 
-    <a class="btn btn-outline d-inline-block mb-1 mb-md-0" href="https://twitter.com/ProbotTheRobot">
+    <a class="btn btn-outline d-inline-block mb-1 mb-md-0 mr-0 mr-md-3" href="https://twitter.com/ProbotTheRobot">
       Follow on Twitter
     </a>
+
+    <a class="btn btn-outline d-inline-block mb-1" href="https://stackoverflow.com/questions/tagged/probot">
+      Probot on Stack Overflow
```
Comment (bkeepers) [inline community.html]: The other ones are "Chat on…", "Contribute on…", "Follow on…"

What about "Ask on Stack Overflow"?

Comment (tcbyrd) [reply community.html]: yeh that works.

**Comment by tcbyrd** [comment]
@bkeepers Any ideas why the tests aren't passing for the remaining few servers? Looks like they're all up and the tests pass on my local system:

```bash
Checking 638 external links...
Ran on 48 files!


HTML-Proofer finished successfully.
```

So I'm getting the same amount of links from the same files from `script/cibuild`, but Travis errors out when pinging a couple of sites hosted on Now.sh

**Comment by bkeepers** [comment]
Welp, that didn't seem to work

**Comment by hiimbex** [comment]
Also seeing travis errors for pinging sites here: https://travis-ci.org/probot/probot.github.io/builds/366531149?utm_source=github_status&utm_medium=notification

**Comment by hiimbex** [comment]
Is it possible that this needs to be added as well? https://github.com/gjtorikian/html-proofer/issues/376#issuecomment-332770021

I tried to test it out but the failing pings didn't let me see the results.

**Comment by bkeepers** [comment]
@probot/maintainers tests for the website are passing again thanks to @tcbyrd and @hiimbex. Going forward let's try to keep them passing.

**Pull request merged**